src/tools/base/BaseBrushTool.js

```diff
@@ -441,16 +461,6 @@ class BaseBrushTool extends BaseTool {
 
     const imageIds = stackToolState.data[0].imageIds;
 
-    const enabledElement = external.cornerstone.getEnabledElement(element);
-
-    const image = enabledElement.image;
-
-    const dim = {
-      xy: image.columns * image.rows,
-      z: image.rows,
-      xyz: image.columns * image.rows * imageIds.length,
-    };
-
```

Comment:
@JamesAPetts did you have plans for this?

Reply:
I plan to tidy up this functionality, but I'm happy for these lines to be deleted as they aren't used. This is something I wrote at the Project Week hackathon and cheekily merged to master.